packages/wallets/keystone/src/adapter.ts

```diff
@@ -102,14 +103,23 @@ export class KeystoneWalletAdapter extends BaseMessageSignerWalletAdapter {
         this.emit('disconnect');
     }
 
-    async signTransaction<T extends Transaction>(transaction: T): Promise<T> {
+    async signTransaction<T extends Transaction | VersionedTransaction>(transaction: T): Promise<T> {
         try {
             const keyring = this._keyring;
             const publicKey = this._publicKey?.toString();
```

Comment:
```suggestion
            const publicKey = this._publicKey;
```
Can you change this line? We no longer need to stringify it.

Reply:
the `publicKey` here is a parameter for keystone keyring and it's type is string so I converted it.